### PR TITLE
Prepare release 3.0.0-pre.1

### PR DESCRIPTION
**Pull request by usergenic**
* [BREAKING] Switched the underlying parser/AST for JavaScript from `espree/estree` to `babylon/babel-types`.  This was needed to support parsing of important platform features such as dynamic imports and moves us closer to supporting TypeScript.
* When printing Warnings, use one-based indexes for lines and columns, as most text editors and other tools do.
* [x] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by usergenic** [comment]
Thanks @rictic 👍

**Comment by rictic** [comment]
The node 6 issue is from an updated dep. I fixed in #766

**Comment by usergenic** [comment]
@rictic ok.  any reason I shouldn't merge master into 3.0 to get the fix?

**Comment by usergenic** [comment]
@rictic oh wait i see thats not merged into master yet.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-<!-- ## Unreleased -->
+## Unreleased
 <!-- Add new, unreleased changes here. -->
 
+## [3.0.0-pre.1] - 2017-11-28
```
Comment (aomarks) [inline CHANGELOG.md]: 11-29

Comment (usergenic) [reply CHANGELOG.md]: k :)

**Pull request merged**